tests/bootstrap-test-env.sh

```diff
@@ -32,7 +32,7 @@ else
 fi
 
 echo "Checking Vagrant installation..."
-vagrant_version=`vagrant --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+"`
+vagrant_version=`vagrant --version | egrep --colour=never -o "[0-9]+\.[0-9]+\.[0-9]+"`
```

Comment:
This is just the first check, but egrep is used other places. Should the other calls be updated?